packages/xrpl/src/models/methods/tx.ts

```diff
@@ -39,8 +40,9 @@ export interface TxRequest extends BaseRequest {
  *
  * @category Responses
  */
-export interface TxResponse<T extends BaseTransaction = Transaction>
-  extends BaseResponse {
+export interface TxResponse<
+  T extends BaseTransaction | PseudoTransaction = Transaction,
```

Comment:
Dont all `PseudoTransaction` options extend BaseTransactions.

Would this work? 

```
T extends BaseTransaction = Transaction | PseudoTransaction,
```

Reply:
I tried that earlier but `UNLModify` does not extend `BaseTransaction`

Reply:
It should be able to.